spec/spec_helper.rb

```diff
@@ -6,14 +6,6 @@
 
 ENV['RACK_ENV'] = 'test'
 
-DB = Sequel.connect(
```

Comment:
We weren't using this - DB is already a global constant whether we're in a test context or not.